buildroot-external/rootfs-overlay/usr/libexec/docker-failure

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ "$SERVICE_RESULT" = "exit-code" ] && [ "$EXIT_STATUS" = "1" ]; then
+    echo "Docker exited with exit status 1, this might be caused by corrupted key.json."
+    size=$(stat -c %s "/etc/docker/key.json")
+    echo "key.json: ${size} bytes"
```

Comment:
Wouldn't it be better to check the contents / check if the json can be parsed?

Reply:
Yeah, I was actually thinking the same. It will still be a heuristic in the end, but I guess there is a quite good chance that there are other errors which lead to the same exit code, where we don't want to remove the `key.json`.